mysql-test/suite/rpl/t/rpl_semi_sync_shutdown_await_ack.inc

```diff
@@ -116,6 +116,9 @@ show status like 'Rpl_semi_sync_master_no_tx';
 --connection server_2
 --eval SET @@GLOBAL.debug_dbug= "$sav_server_2_dbug"
 --eval SET @@GLOBAL.rpl_semi_sync_slave_enabled= 0
+--let $slave_io_errno=1595
+--source include/wait_for_slave_io_error.inc
```

Comment:
The `rpl_semi_sync_shutdown_await_ack` test will be a little more complicated to fix. If you look at the "driver" test, `rpl_semi_sync_shutdown_await_ack.test`, it calls this `.inc` file 4 times, each having different expectations for servers 2 and 3 to error or not.. When `corrupt_queue_event`, is provided to the server, the IO thread will fail with this error (and it can be either server 2 or 3, depending on which test case is running). If a server doesn't have `corrupt_queue_event` I'd imagine this errno would probably be `2003` (lost connection to server) instead. One way to do this would be using `SELECT REGEXP`, something like (using `server_2` as the example, and note it is untested :)):

```
# 2003 is the default..
--let $slave_io_errno=2003

# and if corrupt_queue_event is specified, we should await a relay log error instead
if (`SELECT REGEXP("corrupt_queue_event","$server_2_dbug")`)
{
--let $slave_io_errno=1595
}
```